src/popup/router/components/MenuCarousel.vue

```diff
@@ -1,29 +1,45 @@
 <template>
-  <flickity class="menu-carousel" ref="flickity" :options="flickityOptions">
-    <BoxButton to="/tokens" :text="$t('pages.titles.balances')">
-      <Balances slot="icon" />
-    </BoxButton>
-    <BoxButton to="/send" :text="$t('pages.titles.payments')" class="tour__step7" data-cy="send">
-      <Payments slot="icon" />
-    </BoxButton>
-    <BoxButton to="/tip" :text="$t('pages.titles.tips')" class="tour__step2" data-cy="tip-button">
-      <Tips slot="icon" />
-    </BoxButton>
-    <BoxButton to="/transactions" :text="$t('pages.titles.tx-history')" class="tour__step5">
-      <Activity slot="icon" />
-    </BoxButton>
-    <BoxButton to="/names" :text="$t('pages.titles.names')" class="cell">
-      <Names slot="icon" />
-    </BoxButton>
-    <BoxButton to="/invite" :text="$t('pages.titles.invite')">
-      <Invites slot="icon" />
-    </BoxButton>
-  </flickity>
+  <div class="menu-carousel">
+    <button class="prev" ref="prev" @click="calcTransform(1)"><Arrow /></button>
```

Comment:
store transform in state variable and switch it visibility using `v-if` instead of low-level `ref`